labelCloud/io/segmentations/numpy.py

```diff
@@ -12,7 +12,7 @@ class NumpySegmentationHandler(BaseSegmentationHandler):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
-    def _create_labels(self, num_points: int) -> npt.NDArray[np.int8]:
+    def _create_labels(self, num_points: int, *args, **kwargs) -> npt.NDArray[np.int8]:
```

Comment:
How come you added `*args, **kwargs` here?

Reply:
It is because the parent class has *args and *kwargs therefore it is complained by mypy. Then the question is does it make sense to make *args and **kwargs in the parent class - I think it has better flexiablility if we want to have other behaviors in different handlers. Let me know if you don't agree.

Reply:
I would say we drop `*args, **kwargs` for now and add it only when we need it for a feature.

Reply:
agreed - updated in another PR here [646264a](https://github.com/ch-sa/labelCloud/pull/104/commits/646264a89a866ed68f85c4f167cdb37776cfe067)